### PR TITLE
Add security policies

### PR DESCRIPTION
**Pull request by vestival**

### CODE REVIEW

File: CloudForms-Internals/policies/Policy_DROWN Vulnerability.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        definition: 
+        default: 
+        enabled: 
+      MiqAction:
+        name: d4ea28d8-f051-11e2-97c5-000c297e26aa
+        description: Send Email to Security Team
+        guid: d4ea28d8-f051-11e2-97c5-000c297e26aa
+        action_type: email
+        options:
+          :from: cloudforms_demo@redhat.com
```
Comment (wrichter) [inline CloudForms-Internals/policies/Policy_DROWN Vulnerability.yaml]: Do we need to adjust these email addresses to whichever hailstorm domain this appliance is running in?

**Comment by vestival** [comment]
I will review it

**Comment by wrichter** [comment]
It is trivial to ansibilize and make dynamic, just need to understand if these email addresses are actually used or if those are default values which are overwritten at runtime anyway

**Comment by vestival** [comment]
So please, feel free to ansibilize then :-)

**Pull request merged**